ansible/roles/letsencrypt/tasks/main.yml

```diff
@@ -78,24 +78,16 @@
   - name: Force redirect to https (2/3)
     lineinfile:
       dest: /etc/apache2/sites-available/{{project_name}}.conf
-      line: "RewriteCond %{SERVER_NAME} ={{server_name}}"
-      state: present
-      insertbefore: "</VirtualHost>"
-    sudo: true
-    notify: apache restart
-
-  - name: Force redirect to https (3/3)
-    lineinfile:
-      dest: /etc/apache2/sites-available/{{project_name}}.conf
-      line: "RewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [END,QSA,NE,R=permanent]"
+      line: "RewriteCond %{SERVER_NAME} ={{item}}\nRewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [END,QSA,NE,R=permanent]"
       state: present
       insertbefore: "</VirtualHost>"
     sudo: true
+    with_items: "{{ ([server_name] + server_aliases) }}"
```

Comment:
Yeah I tested it and it works. `server_aliases` is an array, `server_name` is not, so I make `server_name` an array then add on the aliases. We also do this a few lines above in the certbot command to set the domains.